compiler/rustc_target/src/target_features.rs

```diff
@@ -502,6 +502,13 @@ const IBMZ_ALLOWED_FEATURES: &[(&str, Stability, ImpliedFeatures)] = &[
     // tidy-alphabetical-end
 ];
 
+const SPARC_ALLOWED_FEATURES: &[(&str, Stability, ImpliedFeatures)] = &[
+    // tidy-alphabetical-start
+    ("hasleoncasa", Unstable(sym::sparc_target_feature), &[]),
```

Comment:
uhhh, can we call it `leoncasa`?

Reply:
Removing the `has` prefix makes sense to me. Renamed.